src/components/Button/Button.tsx

```diff
@@ -19,7 +19,7 @@ class Button extends UIComponent<any, any> {
 
   public static rules = buttonRules
 
-  public static variables = buttonVariables
+  public static variables: (x) => IButtonVariables = buttonVariables
```

Comment:
I have removed all static rules and variables in #16.  We're breaking that PR down into several smaller PRs, however, there will be no more static rules/variables in any components moving forward.